app/webpacker/components/RegistrationsV2/Register/StripeWrapper.jsx

```diff
@@ -10,9 +10,9 @@ import { fetchJsonOrError } from '../../../lib/requests/fetchWithAuthenticityTok
 import { paymentDenominationUrl } from '../../../lib/requests/routes.js.erb';
 import { useRegistration } from '../lib/RegistrationProvider';
 
-const convertISOAmount = async (amount, currency) => {
+const convertISOAmount = async (competitionId, userId, isoDonation) => {
   const { data } = await fetchJsonOrError(
-    paymentDenominationUrl(amount, currency),
+    paymentDenominationUrl(competitionId, userId, isoDonation),
```

Comment:
Again, the reference to `amount` is missing

Reply:
Fixed